integration_tests/mockApis/nomisMigrationApi.ts

```diff
@@ -613,11 +613,11 @@ const stubHealth = (failures: string = '153'): SuperAgentRequest =>
           'migrationvisitbalance-health': {
             status: 'UP',
             details: {
-              queueName: 'dps-syscon-dev-migration_visitbalance_queue',
+              queueName: 'syscon-devs-dev-migration_visitbalance_queue',
```

Comment:
this was confusing, so changed to names currently returned by `/health` endpoint